integration-templates/bamboohr-basic/actions/update-employee.ts

```diff
@@ -0,0 +1,29 @@
+import type { NangoAction, BamboohrResponseStatus, BamboohrUpdateEmployee } from '../../models';
+
+export default async function runAction(nango: NangoAction, input: BamboohrUpdateEmployee): Promise<BamboohrResponseStatus> {
+    try {
+        if (!input.id) {
+            throw new nango.ActionError({
+                message: 'id is a required field'
+            });
+        }
+        const { id, ...postData } = input;
+
+        const response = await nango.post({
+            endpoint: `/v1/employees/${input.id}`,
+            data: postData
+        });
+
+        return {
+            status: response.statusText
+        };
+    } catch (error: any) {
+        const messageHeader = error.response.headers['x-bamboohr-error-message'];
```

Comment:
this is why I was saying it's "dangerous", you are throwing an ActionError but you assume it will be an AxiosError leading to more error

Reply:
Thanks. Updated to check for the object, but as mentioned in the other PR the part I want to capture is the bamboo specific error.